tsl/test/expected/continuous_aggs_ddl-11.out

```diff
@@ -943,3 +939,107 @@ SELECT chunk_table, ranges FROM chunk_relation_size('drop_chunks_table');
  _timescaledb_internal._hyper_12_19_chunk | {"[50,60)"}
 (2 rows)
 
+--test drop_chunks with cascade_to_materialization set to true (github 1644)
+-- This checks if chunks from mat. hypertable are actually dropped 
+-- and deletes data from chunks that cannot be dropped from that mat. hypertable.
+SELECT format('%s.%s', schema_name, table_name) AS drop_chunks_mat_tablen
+    FROM _timescaledb_catalog.hypertable, _timescaledb_catalog.continuous_agg
+    WHERE _timescaledb_catalog.continuous_agg.raw_hypertable_id = :drop_chunks_table_nid
+        AND _timescaledb_catalog.hypertable.id = _timescaledb_catalog.continuous_agg.mat_hypertable_id \gset
+SELECT drop_chunks(table_name => 'drop_chunks_table', older_than=>integer_now_test2() + 200, cascade_to_materializations => true);
+               drop_chunks                
+------------------------------------------
+ _timescaledb_internal._hyper_12_19_chunk
+ _timescaledb_internal._hyper_12_15_chunk
+(2 rows)
+
+SELECT count(c) FROM show_chunks('drop_chunks_table') AS c;
+ count 
+-------
+     0
+(1 row)
+
+SELECT count(c) FROM show_chunks(:'drop_chunks_mat_tablen') AS c;
+ count 
+-------
+     0
+(1 row)
+
+SELECT set_chunk_time_interval('drop_chunks_table', 10);
+ set_chunk_time_interval 
+-------------------------
+ 
+(1 row)
+
+SELECT set_chunk_time_interval(:'drop_chunks_mat_tablen', 20);
```

Comment:
I assume it is not common to set the chunk_time_interval on the materialization table? Again, would be good to get some hints on what you're trying to achieve here?

I guess it has something to do with trying to setup misaligned chunks between the raw hypertable and mat. hypertable?

Reply:
Added a comment before the drop_chunks call.
The default mat. hypertable chunk size is 10 times the raw hypertable's chunk. I am trying to make sure atleast 1 chunk from the mat. hypertable will get dropped without having to insert too much data. Also want data from the other chunk to get deleted.